src/main/server.ts

```diff
@@ -1,26 +1,64 @@
 #!/usr/bin/env node
 
-import { app } from './app'
+import * as tsConfigPaths from 'tsconfig-paths'
+import { cpus } from 'os'
+import * as express from 'express'
 import * as fs from 'fs'
 import * as path from 'path'
 import * as https from 'https'
+import * as cluster from 'cluster'
 
+const tsConfig = require('../../tsconfig.json')
+const numberOfCores = cpus().length
 const port: number = parseInt(process.env.PORT, 10) || 3000
-
 const logger = require('@hmcts/nodejs-logging').getLogger('server')
 
-if (app.locals.ENV === 'development' || app.locals.ENV === 'dockertests') {
+function bootstrapTypeScriptPaths (tsConfig: any) {
```

Comment:
Would be great to have it (alongside `require('../../tsconfig.json')`) in separate file e.g. `ts-paths-bootstrap.ts`. Then we could do `import './ts-paths-bootstrap.ts'` at the beginning of the file.

The reason of the change is to separate concerns.